cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -207,17 +180,44 @@ macro(configure_sdk_unix
   endif()
 
   foreach(arch ${architectures})
+    if("${prefix}" STREQUAL "ANDROID")
+      if("${arch}" STREQUAL "armv7")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "arm-linux-androideabi")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "arm")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-arm")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "armv7-none-linux-androideabi")
+      elseif("${arch}" STREQUAL "aarch64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "aarch64-linux-android")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "aarch64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-arm64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "aarch64-unknown-linux-android")
```

Comment:
`unknown` or `none`? For consistency with ARMv7, and to not confuse people later, they should be the same (the documentation uses `none`).

Reply:
Technically, the vendor should be `unknown`, not `none`.